src/components/widgets/Number.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import Header from './Header'
+import "../../styles/widget/number.css"
+import "../../styles/widget/widget.css"
+
+class Number extends Component {
+
+  calculateProgress( mainStat, goal ) {
+    const progressNum = + ( ( mainStat / goal ) * 100 ).toFixed(2)
+    return ( progressNum ).toString().concat('%')
+  }
+
+  render() {
+
+    const { title, x, y, mainStat, goal } = this.props
+    const positionStyle = { top: x, left: y }
+    const progressString = this.calculateProgress( mainStat, goal )
+    const isComplete = mainStat / goal >= 1
+
+    return (
+      <article>
+        <div className="widget widget-width-1 widget-height-1" style={ positionStyle }>
+          <Header title={ title }/>
+          <div className="widget-body stat-wrapper">
+            <div className="main-stat">{ mainStat }</div>
+            <div className="goal-progress-wrapper">
+              <div
+                className={ isComplete
+                  ? "progress-stat progress-complete"
+                  : "progress-stat" }
+                >
+                { progressString }
+              </div>
+              <div
+                className={ isComplete
+                  ? "goal-stat goal-complete"
+                  : "goal-stat" }
+                >{ goal }</div>
+            </div>
+            <div className={ isComplete
+                ? "progress-bar goal-complete"
+                : "progress-bar" }
+              >
+              <span style={ { width: isComplete ? '100%' : progressString } }></span>
```

Comment:
This check will prevent the progress bar from rendering past 100% width. Could you clarify, how would we use Math.ceil instead of this check?